.config/tsaoptions.json

```diff
@@ -0,0 +1,15 @@
+{
+    "instanceUrl": "https://microsofthealth.visualstudio.com",
+    "TSADocumentation": "https://aka.ms/OBTSA",
+    "projectName": "Health",
+    "areaPath": "Health\\HLS Interop\\Aegean",
+    "notificationAliases": [
+        "duburson@microsoft.com",
```

Comment:
@dustinburson is there an different alias that we should use here? We have `Aegean Support <medtechsupport@microsoft.com>` and `HLS_DP_InterOp_CS@microsoft.com`